packages/mask/src/social-network/ui.ts

```diff
@@ -120,6 +121,12 @@ export async function activateSocialNetworkUIInner(ui_deferred: SocialNetworkUI.
                 MaskMessages.events.currentPersonaIdentifier.on,
                 signal,
             )
+            const allPersonaSub = createSubscriptionFromAsync(
+                () => Services.Identity.queryOwnedPersonaInformation(true),
```

Comment:
Do you sure you want to include the unintialized personas?

Reply:
I need nickname and avatar url of current persona but currentPersona from ``` Services.Settings.getCurrentPersonaIdentifier``` doesn't include these

Reply:
I mean, `Services.Identity.queryOwnedPersonaInformation(true)` and `Services.Identity.queryOwnedPersonaInformation(false)` are different.